spec/views/layouts/application_spec.rb

```diff
@@ -3,19 +3,14 @@
 require 'rails_helper'
 
 describe 'layouts/application' do
-  context 'when current user is present' do
```

Comment:
we removed the conditional logic from the navbar in #411. So folks see these links whether there's a "current user" or not.